src/veh_type.cpp

```diff
@@ -79,6 +79,7 @@ static const std::unordered_map<std::string, vpart_bitflags> vpart_bitflag_map =
     { "ENABLED_DRAINS_EPOWER", VPFLAG_ENABLED_DRAINS_EPOWER },
     { "WASHING_MACHINE", VPFLAG_WASHING_MACHINE },
     { "FLUIDTANK", VPFLAG_FLUIDTANK },
+    { "HEATER", VPFLAG_HEATER },
```

Comment:
Shouldn't it be `E_HEATER` as in jsons above?

Reply:
Also it is not being used anywhere in the code. Do we need it at all?

Reply:
You're right it should have been changed to E_HEATER, missed this one.  It gets used in map.cpp:4466.  Thanks, will fix

Reply:
Oh wait, I see what you mean now, the VPFLAG_HEATER doesn't actually get used anywhere so you're right it's not necessary.  Good call, thanks!.